plugins/package-managers/cocoapods/src/main/kotlin/CocoaPods.kt

```diff
@@ -271,10 +279,27 @@ private fun getPackageReferences(podfileLock: File): Set<PackageReference> {
             dependencies = dependenciesForName[name].orEmpty().mapTo(mutableSetOf()) { createPackageReference(it) }
         )
 
-    return root.get("DEPENDENCIES").mapTo(mutableSetOf()) { node ->
+    root.get("DEPENDENCIES").mapTo(references) { node ->
         val name = node.textValue().substringBefore(" ")
         createPackageReference(name)
     }
+
+    root.get("CHECKOUT OPTIONS")?.fields()?.asSequence()?.forEach {
+        val checkout = it.value as ObjectNode
+        val url = checkout[":git"].textValueOrEmpty()
```

Comment:
As currently only `git` is implemented here, how about skipping the iteration in case there is no key `:git` ?